app/managers.py

```diff
@@ -0,0 +1,46 @@
+import sqlite3
+from app.models import Actor
+
+
+class ActorManager:
+    def __init__(self) -> None:
+        self._connection = sqlite3.connect("cinema.sqlite")
```

Comment:
Consider adding a method to close the database connection when it's no longer needed. This will help prevent resource leaks and ensure that the connection is properly managed.